app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -409,7 +409,8 @@ class BookmarkFragment : LibraryPageFragment<BookmarkNode>(), UserInteractionHan
 
     private fun updatePendingBookmarksToDelete(selected: Set<BookmarkNode>) {
         pendingBookmarksToDelete.addAll(selected)
-        val bookmarkTree = sharedViewModel.selectedFolder!! - pendingBookmarksToDelete
+        val selectedFolder = sharedViewModel.selectedFolder ?: return
```

Comment:
do we need to log the NPE, or is this info we don't really care about?

Reply:
Thanks for bringing that up I think we should investigate if we should refactor how we deal with bookmarks, I noticed there are other usages of `selectedFolder!!` that could cause NPE, I don't have the whole context around why it was built in that way, for now I think we should stop the crash until we do the refactor,  I filed https://github.com/mozilla-mobile/fenix/issues/24896 for investigating.